//
// Created by Екатерина Акулова on 28.10.2022.
//

#include "Exeption.hpp"

Exeption::Exeption()
{
    std::cout <<"Default Constructor for Exeption called " << std::endl;
};
Exeption::~Exeption()
{
    std::cout <<"Destructor for Exeption called " << std::endl;
};
